src/OpenTK.Windowing.Desktop/NativeWindow.cs

```diff
@@ -602,7 +684,11 @@ public unsafe NativeWindow(NativeWindowSettings settings)
 
             RegisterWindowCallbacks();
 
-            IsFocused = settings.StartFocused;
+            _isFocused = settings.StartFocused;
+            if (settings.StartFocused)
+            {
+Focus();
```

Comment:
StyleCop told me "Braces and tabs must be used correctly" until I used them incorrectly, and then it stopped whining.